app/models/concerns/rubygem_searchable.rb

```diff
@@ -2,14 +2,32 @@ module RubygemSearchable
   extend ActiveSupport::Concern
 
   included do
-    include Elasticsearch::Model
-
-    index_name "rubygems-#{Rails.env}"
-
-    delegate :index_document, to: :__elasticsearch__
-    delegate :update_document, to: :__elasticsearch__
+    searchkick index_name: "rubygems-#{Rails.env}", settings: {
```

Comment:
You may want to disable automatic syncing (set `callbacks: false`) if calling `reindex` manually.

Reply:
I'll check on this to make behaviour 1:1. I'm not sure how it was set before.

Reply:
updated, thanks for the hint